components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -82,6 +83,13 @@ sealed class SystemAction : BrowserAction() {
     ) : SystemAction()
 }
 
+/**
+ * Updating the [BrowserState] to reflect app [Locale] changes.
+ *
+ * @property locale the updated [Locale]
+ */
+data class UpdateLocaleAction(val locale: Locale) : BrowserAction()
```

Comment:
As in `BrowserState` support null values I think this one should be null-able too, what do you think?